app/uk/gov/hmrc/views/ViewConfig.scala

```diff
@@ -45,6 +45,7 @@ class ViewConfig @Inject()(val configuration: Configuration,
   lazy val lockoutPeriodEn: String = configuration.getOptional[String]("lockout.period.en").getOrElse("24 hours")
   lazy val lockoutPeriodCy: String = configuration.getOptional[String]("lockout.period.cy").getOrElse("24 awr")
   lazy val isLocal: Boolean = configuration.getOptional[Boolean]("isLocal").getOrElse(false)
+  lazy val findHelplineFrontendUrl: String = configuration.getOptional[String]("helpline-frontend.host").getOrElse{""} + "/helpline/which-service-are-you-trying-to-access"
```

Comment:
I think the argument to getOrElse is usually bounded by open brackets, rather than curly ones.

It does seem to work though.

Reply:
This is updated